src/jimgw/single_event/transforms.py

```diff
@@ -24,6 +26,131 @@
 )
 
 
+@jaxtyped(typechecker=typechecker)
+class UniformInComponentMassSecondaryMassTransform(ConditionalBijectiveTransform):
```

Comment:
Is this a transform or a prior? Also, is this used in the testing script?

Reply:
It is a transform